samples/bot-conversation/python/.vscode/launch.json

```diff
@@ -6,9 +6,6 @@
             "type": "msedge",
             "request": "launch",
             "url": "https://teams.microsoft.com/l/app/${{local:TEAMS_APP_ID}}?installAppPackage=true&webjoin=true&${account-hint}",
-            "cascadeTerminateToConfigurations": [
-                "Attach to Local Service"
```

Comment:
I remember the purpose of this configuration is to stop a task when user closes the browser. Should we add `Python: Run App Locally` here? We can test whether the Python process will terminate with current configuration first.